build.gradle.kts

```diff
@@ -1,9 +1,15 @@
 plugins {
     java
     `java-library`
+    jacoco
 }
 
 allprojects {
+    apply(plugin = "java")
+    if (System.getenv("JACOCO") == "true") {
```

Comment:
nit: Using gradle project properties seem to be more appropriate (scope of the var is narrowed down for Gradle usage exclusively). Project properties can be passed with the `-P` option but there is a way to pass them through env vars as well: https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties

Reply:
Fully agree - it seemed though easier on maintainers to follow exactly what's done in EDC